test/pylib/util.py

```diff
@@ -131,6 +130,23 @@ async def find_host():
     return await wait_for(find_host, deadline)
 
 
+async def gather_wrapper(*awaitables: Awaitable, return_exceptions: bool = False):
+    """
+    Wrapper around asyncio will pass awaitables to asyncio.gather() with parameter return_exceptions=True.
+    When the function will get return_exceptions=False, it will raise the first exception in from the awaitables.
+    When return_exceptions=True, it will return the results of the awaitables and the caller needs to handle exceptions
+    """
+    if return_exceptions:
+        return await asyncio.gather(*awaitables, return_exceptions=True)
+    else:
+        results = await asyncio.gather(*awaitables, return_exceptions=True)
+        for result in results:
+            if isinstance(result, BaseException):
+                raise result from None
```

Comment:
How is "raise from None" different from just "raise" in this case?

Reply:
This will allow us to exclude gather_wrapper from the traceback since it just rethrowing the exception. 
The method will throw only the first exception even if there will be more of them. Do you think it will be beneficial to gather all exceptions to the list and throw custom exceptions with that list?